symengine/complex_double.h

```diff
@@ -67,50 +63,53 @@ class ComplexDouble : public Number {
     /*! Add ComplexDoubles
      * \param other of type Integer
      * */
-    RCP<const Number> addcomp(const Integer &other) const {
+    RCP<const Number> addcomp(const Integer &other) const
+    {
         return make_rcp<const ComplexDouble>(i + mp_get_d(other.i));
     }
 
```

Comment:
Or we can stick to not using single line functions at all. That will keep things much simpler.